controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -618,6 +618,23 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
         LOG(DEBUG) << "WSC config_data: tear down";
     }
 
+    // The MAC address in the config data is tricky... According to "Wi-Fi Simple Configuration
+    // Technical Specification v2.0.4", section 7.2.2 "Validation of Configuration Data" the MAC
```

Comment:
We should refer to v2.0.6 which is the WSC specification referred to by the EasyMesh Specification, see section 2 
>[5]	Wi-Fi Simple Configuration Technical Specification 2.0.6 (https://www.wi-fi.org/members/certification-programs )

Reply:
Meh, I don't have that version of the document and I can't access the WFA site. Can you confirm that the references are still correct?

And then I should try and get version 2.0.6...

Reply:
The references are still correct.